home.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Apertium</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+    <h1>Welcome to apertium-apy</h1>
+    <div>Please follow the link to find apertium-apy docs.</div>
+    <div><a href="http://wiki.apertium.org/wiki/Apertium-apy">Click to follow</a></div>
```

Comment:
"Click here links" are a UX faux pas. I prefer something more even more simple like:

```
<h1>Apertium APy</h1>
<h3><a href="...">Documentation</a><h3>
<h3><a href="...">Source</a></h3>
```

Reply:
I can do that